### DISCUSSION TITLE
WindowBuilder not showing in Eclipse after downloading

### DISCUSSION THREAD

**Discussion by Typheta** [eclipse/windowbuilder#341] [Q&A]
So Ive been trying to download WindowBuilder for Eclipse but it won't work. I have tried downloading straight from the marketplace and from Eclipse.org but nothing works. When it installs and restarts, I am unable to open or create a file with WindowBuilder.

I have deleted and reinstalled WindowBuilder several times. I have tried different versions, but nothing works. I have deleted my meta data folder but that didn't work either.

In  Eclipse-workspace > .metadata >.plugins there are no WindowBuilder files but it says it is installed when I check my marketplace. I am on the latest version of Eclipse.

 Everything is up to date (in the marketplace picture it says update after I delete it and reinstall it. I have already updated it and still nothing worked).
<img width="1440" alt="Screen Shot 2022-10-20 at 11 47 03 AM" src="https://user-images.githubusercontent.com/73000873/196998409-6ecc64f4-9cc7-4fef-9bb9-0875da8e5de0.png">

<img width="1440" alt="Screen Shot 2022-10-20 at 11 47 55 AM" src="https://user-images.githubusercontent.com/73000873/196998435-fb7791

<img width="1440" alt="Screen Shot 2022-10-20 at 11 48 23 AM" src="https://user-images.githubusercontent.com/73000873/196998454-d204a095-7322-447d-87b4-e39b0cc569f7.png">
48-2a86-4025-aca9-05a0e1a0068a.png">

**Comment by wimjongman** [comment]
It is not installed or not installed correctly.

Press CTRL+3 and find 'copy configuration data to clipboard'.  Put that in a file and attach the file here.

**Reply by Typheta** [reply]
Ive commented it

**Comment by Typheta** [comment]
Here is the data configuration 
[Data config.txt](https://github.com/eclipse/windowbuilder/files/9852019/Data.config.txt)

**Comment by wimjongman** [comment]
No sign of WB in the config, so it is not installed.

Have you unpacked Eclipse with the root account? Maybe you can make a video of the whole process and attach that.
